src/Common/src/Interop/Windows/Crypt32/Interop.certificates_imports.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
@bartonjs I'm fine with changing the name of the file from "Interop.certificates_imports.cs" to something else.  What do you suggest?  Just have it be "Interop.certificates.cs".  Will that help devs know that it just contains DLL-Import statements?  The other file has just the structs, const's, etc. in "Interop.certificates_types.cs".